SPECS-EXTENDED/ocaml-calendar/ocaml-calendar.spec

```diff
@@ -41,50 +52,44 @@ developing applications that use %{name}.
 
 
 %prep
-%setup -q -n calendar-%{version}
-%patch 1 -p1
+%autosetup -n calendar-%{version} -p1
 
 
 %build
-./configure --libdir=%{_libdir}
-make
-make doc
-
-mv TODO TODO.old
-iconv -f iso-8859-1 -t utf-8 < TODO.old > TODO
+%dune_build
 
 
 %install
-export DESTDIR=$RPM_BUILD_ROOT
-export OCAMLFIND_DESTDIR=$RPM_BUILD_ROOT%{_libdir}/ocaml
-mkdir -p $OCAMLFIND_DESTDIR $OCAMLFIND_DESTDIR/stublibs
-make install
+%dune_install
 
 
-%files
-%doc CHANGES README TODO LGPL COPYING
-%{_libdir}/ocaml/calendar
-%ifarch %{ocaml_native_compiler}
-%exclude %{_libdir}/ocaml/calendar/*.cmx
+%if %{with tests}
+%check
+%dune_check
 %endif
-%exclude %{_libdir}/ocaml/calendar/*.mli
 
 
-%files devel
-%doc CHANGES README TODO LGPL COPYING calendarFAQ-2.6.txt doc/*
-%ifarch %{ocaml_native_compiler}
-%{_libdir}/ocaml/calendar/*.cmx
-%endif
-%{_libdir}/ocaml/calendar/*.mli
+%files -f .ofiles
+%doc CHANGES README.md TODO
+%license LGPL COPYING
+
+
+%files devel -f .ofiles-devel
+%doc CHANGES README.md TODO calendarFAQ-2.6.txt
+%license LGPL COPYING
 
 
 %changelog
+* Fri Dec 20 2024 Durga Jagadeesh Palli <v-dpalli@microsoft.com> - 3.0.0-1
+- Update to 3.0.0
+- License verified
+
 * Thu Oct 14 2021 Pawel Winogrodzki <pawelwi@microsoft.com> - 2.04-29
 - Switching to using full number for the 'Release' tag.
 - Initial CBL-Mariner import from Fedora 32 (license: MIT).
 
 * Thu Feb 27 2020 Richard W.M. Jones <rjones@redhat.com> - 2.04-28.1
-- OCaml 4.10.0 final (Fedora 32).
+-- OCaml 4.10.0 final (Fedora 32).
```

Comment:
Older changelogs shouldn't be modified unless absolutely necessary (something like typo). If you are importing Fedora SPEC file, please use the changelog from the original spec file itself.